### PR TITLE
WDP240901-5

### PR DESCRIPTION
**Pull request by dave8git**
https://projects.kodilla.com/secure/RapidBoard.jspa?rapidView=239&projectKey=WDP240901&view=detail&selectedIssue=WDP240901-5

### CODE REVIEW

**Review by patrykbn — APPROVED** [review]

**Review by patrykbn — APPROVED** [review]
Everything looks good to me

File: src/components/common/ProductBox/ProductBox.js [hunk 1]
```diff
@@ -11,8 +11,9 @@ import {
 import { faStar as farStar, faHeart } from '@fortawesome/free-regular-svg-icons';
 import Button from '../Button/Button';
 
-const ProductBox = ({ name, price, promo, stars }) => (
+const ProductBox = ({ name, price, promo, stars, compare, favorite }) => (
   <div className={styles.root}>
+    {console.log('compare', compare)}
```
Comment (matwjablonski) [inline src/components/common/ProductBox/ProductBox.js:16]: remove console log

File: src/components/common/ProductBox/ProductBox.js [hunk 2]
```diff
@@ -39,10 +40,16 @@ const ProductBox = ({ name, price, promo, stars }) => (
     <div className={styles.line}></div>
     <div className={styles.actions}>
       <div className={styles.outlines}>
-        <Button variant='outline'>
+        <Button 
+          className={favorite ? styles.activeColor : undefined }
```
Comment (matwjablonski) [inline src/components/common/ProductBox/ProductBox.js:44]:
```js
favorite && styles.activeColor
```

File: src/components/common/ProductBox/ProductBox.js [hunk 3]
```diff
@@ -39,10 +40,16 @@ const ProductBox = ({ name, price, promo, stars }) => (
...
+          variant='outline'
+          >
           <FontAwesomeIcon icon={faHeart}>Favorite</FontAwesomeIcon>
         </Button>
-        <Button variant='outline'>
+        <Button 
+          className={compare ? styles.activeColor : undefined }
```
Comment (matwjablonski) [inline src/components/common/ProductBox/ProductBox.js:50]: as above

File: src/components/common/ProductBox/ProductBox.module.scss [hunk 4]
```diff
@@ -34,6 +34,10 @@
     }
   }
 
+  .activeColor {
```
Comment (matwjablonski) [inline src/components/common/ProductBox/ProductBox.module.scss:37]: maybe name could be just `active` and it should be understandable

**Review by matwjablonski — CHANGES_REQUESTED** [review]

File: src/components/common/ProductBox/ProductBox.js (hunk 1 above)
Comment (dave8git) [reply src/components/common/ProductBox/ProductBox.js]: console.log removed.

File: src/components/common/ProductBox/ProductBox.js (hunk 3 above)
Comment (dave8git) [reply src/components/common/ProductBox/ProductBox.js]: Fixed logice.

File: src/components/common/ProductBox/ProductBox.module.scss (hunk 4 above)
Comment (dave8git) [reply src/components/common/ProductBox/ProductBox.module.scss]: Changed according to recomendation.

File: src/redux/initialState.js [hunk 5]
```diff
@@ -26,6 +28,8 @@ const initialState = {
       stars: 2,
       promo: 'sale',
       newFurniture: true,
+      favorite: true,
+      compare: false,
```
[the remark this answers is not in the document]
Comment (dave8git) [reply src/redux/initialState.js:32]: fixed

File: src/components/common/ProductBox/ProductBox.js [hunk 6]
```diff
@@ -10,7 +10,7 @@ import {
 import { faStar as farStar, faHeart } from '@fortawesome/free-regular-svg-icons';
 import Button from '../Button/Button';
 
-const ProductBox = ({ name, price, promo, stars, backgroundPhoto, prevPrice }) => (
+const ProductBox = ({ name, price, promo, stars, backgroundPhoto, compare, favorite, prevPrice }) => (
```
[the remark this answers is not in the document]
Comment (dave8git) [reply src/components/common/ProductBox/ProductBox.js:13]: proptypes added.

File: src/redux/initialState.js [hunk 7]
```diff
@@ -70,6 +80,9 @@ const initialState = {
       stars: 2,
       promo: 'sale',
       newFurniture: true,
+      favorite: false,
+      compare: false,
+      newFurniture: false,
```
[the remark this answers is not in the document]
Comment (dave8git) [reply src/redux/initialState.js]: deleted unnecessary line.

**Review by matwjablonski — APPROVED** [review]

**Pull request merged**